builder-support/debian/authoritative/debian-buster/tests/control

```diff
@@ -1,25 +1,29 @@
 Tests: smoke-bind
 Depends: dnsutils,
+         pdns-tools,
          pdns-backend-bind,
          pdns-server
 Restrictions: needs-root
 
 Tests: smoke-mysql
 Depends: dnsutils,
```

Comment:
i tried to keep the dependencies sorted by name

Reply:
makes sense!

Reply:
oh, do we even still need dnsutils?

Reply:
probably not!

Reply:
ok, will test without

Reply:
PR amended!